src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Fixed a race condition happening in simultaneous OpenMetrics and PlainText
+  requests which could cause malformed response
+  ([#5517](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5517))
```

Comment:
@hez2010 Do we need a corresponding entry in the HttpListener project or is it not subject to the issue?

https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Prometheus.HttpListener/CHANGELOG.md